jest.config.js

```diff
@@ -2,8 +2,16 @@ const { jestConfig } = require('@salesforce/sfdx-lwc-jest/config');
 module.exports = {
     ...jestConfig,
     moduleNameMapper: {
+        "^c/psElevateTokenHandler$": "<rootDir>/tests/__mocks__/psElevateTokenHandler",
```

Comment:
This allows us to have a mocked psElevateTokenHandler inside tests. The whole class is replaced with this implementation, which is as close to actual as possible minus using the iframe's postMessage.